css/styles.css

```diff
@@ -1,3 +1,86 @@
 body {
-    background-color: aqua;
+    background-color: #232423;
+    color: #E9B73B;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+}
+
+.title{
+    text-align: center;
+    color: #E9B73B;
+    font-size: large;
+    position: absolute;
+    right: 0;
+    left: 0;
+    margin: 0 auto;
+    top: 5%;
+}
+
+.enteringTaskField{
+    box-sizing: border-box;
+}
+.todoField{
+    background-color: #FFFFFF; 
+    color: #A1A1A1;
+    border: solid 2px;
+    border-color: #E9B73B; 
+    border-radius: 5px;
+    width: 40%;
+    padding: 10px;
+    font-family: monospace;
+    font-size: medium;
+    position: absolute;
+    right: 10%;
+    left: 0;
+    margin: 0 auto;
+    top: 25%;
+}
+
+.addTaskBtn{
+    background-color: #E9B73B; 
+    color: #232423;
+    border: none;
+    border-radius: 5px;
+    width: 10%;
+    padding: 10px;
+    font-family: monospace;
+    font-size: large;
+    position: absolute;
+    right: 0;
+    left: 45%;
+    margin: 0 auto;
+    top: 25%;
+}
+
+#tasks{
+    background-color: #232423; 
+    color: #E9B73B;
+    border: solid 2px;
+    border-color: #E9B73B; 
+    border-radius: 5px;
+    width: 40%;
+    padding: 10px;
+    font-family: monospace;
+    font-size: medium;
+    position: absolute;
+    right: 10%;
+    left: 0;
+    margin: 0 auto;
+    top: 35%;
+}
+
+
+footer{
```

Comment:
Please, consider adding different font-family attributes for the whole document.